README.md

```diff
@@ -92,7 +93,7 @@ stern supports outputting custom log messages.  There are a few predefined
 templates which you can use by specifying the `--output` flag:
 
 | output    | description                                                                                           |
-|-----------|-------------------------------------------------------------------------------------------------------|
+| --------- | ----------------------------------------------------------------------------------------------------- |
```

Comment:
Do not include any style changes that are not directly related to the function.

Reply:
Copy that. That's a VSC plugin that auto-formats on save. I'll fix this.